README.md

```diff
@@ -1228,6 +1230,40 @@ There would also be logging to the standard out stream as well.
 
 The `cromwell.<date>.log` file contains an aggregate of every log message, while the `workflow.<uuid>.log` files contain only log messages that pertain to that particular workflow.
 
+# Call Caching
+
+Call Caching allows Cromwell to re-use the outputs of a call if the exact same call is about to be run again. 
+**Call Caching is disabled by default.**
+
+To enable Call Caching, add the following to your Cromwell configuration:
+
+```
+call-caching {
+  enabled = true
+}
+```
+
+Workflow options can be used to tune cromwell's caching behaviour for a particular workflow:
+
+```
+{
+  "write-to-cache": false
+}
+```
+
+If `write-to-cache` is set to false, the calls performed by this workflow will not be cached, therefore their output won't be used by subsequent identical calls.
+  
+```
+{
+  "read-from-cache": false
```

Comment:
as I was documenting some of the workflow options, I realized that `read-from-cache` and `write-to-cache` are odd-balls because they use `-` as separators whereas everything else uses `_`.  Is anybody not okay with using `read_from_cache` and `write_to_cache`?